features/analytics.feature

```diff
@@ -0,0 +1,11 @@
+Feature: Analytics
+  In order to know how Citygram is performing
+  As a user
+  I want to see subscription analytics
+
+  Scenario: Viewing Analytics
+    Given there are 200 Email Subscriptions
+    And there are 150 SMS Subscriptions
+    When I visit '/analytics'
+    Then I should see '200 Email Subscriptions'
+    And I should see '150 SMS Subscriptions'
```

Comment:
Hey @noneck, check out what we're building, this will be useful for publicity.

Remind me to put this in the newsletter that I definitely need to write up!